manifests/00-custom-resource-definition.yaml

```diff
@@ -1,24 +1,33 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  creationTimestamp: null
   name: ingresscontrollers.operator.openshift.io
 spec:
   group: operator.openshift.io
   names:
     kind: IngressController
-    listKind: IngressControllerList
     plural: ingresscontrollers
-    singular: ingresscontroller
-  scope: Namespaced
```

Comment:
@ironcladlou why is `IngressController` no longer namespaced?

Reply:
Idiosyncrasy (and possible bug with) controller-tools (https://github.com/kubernetes-sigs/controller-tools/issues/214). However, not a problem for us because the scope defaults to namespaced. This explicit configuration may return later depending on how controller-tools addresses the issue.

Reply:
Thanks for the explanation and issue reference.